src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -7445,6 +7445,12 @@
             "description": "Non-default value of setting."
           }
         },
+        "metrics:useStringIndices": {
```

Comment:
nit: what if we rename `metrics:useStringIndices` -> `metrics:allowStringIndices` . I think semantically this is more correct